### PR TITLE
Sterilizes the codebase - Pt.8 Removes Felinid Language

### PR DESCRIPTION
**Pull request by JamieD1**
Removes Felinid Language

:cl:  
rscdel: Removes Felinid Language
/:cl:

### CODE REVIEW

**Comment by N3D6** [comment]
Unironically the worst change here because literally every race but felinids gets a snowflake language

**Review by redmoogle — CHANGES_REQUESTED** [review]
remove `#undef CAT`

**Comment by ShadowDeath6** [comment]
Felinids aren't their own race according to the health analyzer (felinid human), although removing snowflake languages from races aside from ashwalkers would probably improve the game as well

**Comment by VaelophisNyx** [comment]
what happened to "improve not remove" hmm?

**Comment by Smudged** [comment]
Felinids especially being a pay to play race should keep their features.

**Comment by ynot01** [comment]
i cant look at the code rn but make sure they have english

**Comment by VaelophisNyx** [comment]
Unique languages are good and this PR is a step too far tbh; if this is an attempt to push felinids back into the human-box, it's a poorly done attempt.

**Comment by GraveHat** [comment]
> Felinids aren't their own race according to the health analyzer (felinid human), although removing snowflake languages from races aside from ashwalkers would probably improve the game as well

That is a stretch of a reason to remove an entire language if you aren't planning on axing the rest of them. Although they are considered human, they're still a different option you could choose from human if you were a donater.

**Comment by Cartlord** [comment]
Giving donors their own language is just making the metagang official

**Comment by N3D6** [comment]
> Giving donors their own language is just making the metagang official

Brain dead take

if felinids can’t have a species specific language remove it from every other race so the other races cant metagang

**Comment by SomeguyManperson** [comment]
Don't think this is required

**Comment by Hislyna** [comment]
Just alter what non felinids can hear, alongside some of the descriptions for learning the language and I think everyone would appreciate it more.

File: code/modules/mob/living/carbon/human/species_types/felinid.dm [hunk 1]
```diff
@@ -14,7 +14,6 @@
 	mutanttail = /obj/item/organ/tail/cat
 	changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_PRIDE | MIRROR_MAGIC | RACE_SWAP | ERT_SPAWN | SLIME_EXTRACT
 	swimming_component = /datum/component/swimming/felinid
-	species_language_holder = /datum/language_holder/felinid
```
Comment (Anvilman6) [inline code/modules/mob/living/carbon/human/species_types/felinid.dm:17]: species_language_holder = /datum/language_holder/english

**Review by Anvilman6 — CHANGES_REQUESTED** [review]
If they're becoming more in line with actually being Human, then they get the Human language

File: code/modules/mob/living/carbon/human/species_types/felinid.dm (hunk 1 above)
Comment (Marmio64) [reply code/modules/mob/living/carbon/human/species_types/felinid.dm:17]: bri'ish catperson

**Comment by ghost** [comment]
Remove felenids from the game

**Comment by JamieD1** [comment]
How about we make the language less Nya

File: code/modules/language/felinid.dm [hunk 2]
```diff
@@ -1,13 +0,0 @@
-/datum/language/felinid
-	name = "Felinid"
-	desc = "A language devised by felinids to secretly plot against regular humans"
-	exclaim_verb = "nyas"
-	key = "f"
-	flags = TONGUELESS_SPEECH
-	space_chance = 90
-	syllables = list(
-		"meow", "nya", "nyam","mm","mrow","mmrrrr"
```
Comment (Anvilman6) [inline code/modules/language/felinid.dm:9]: "meow", "mow", "rrrmow","mm","mrow","mmrrrr"

**Review by Anvilman6 — CHANGES_REQUESTED** [review]
Okay

File: code/modules/language/felinid.dm (hunk 2 above)
Comment (VaelophisNyx) [reply code/modules/language/felinid.dm:9]: "mm" doesn't make a lot of sense as a ""cat"" language thing tbh

**Comment by Hislyna** [comment]
/datum/language/felinid
    name = "Felinid"
    desc = "A language devised by felinids to collaborate and hunt more efficiently"
    exclaim_verb = "whistles"
    key = "f"
    flags = TONGUELESS_SPEECH
    space_chance = 90
    syllables = list(
        "hisss", "fweet", "mia","meow","miaow","mmrrrr"
    )
    icon_state = "felinid"
    default_priority = 90
    sing_verb = "miaows"



var/datum/language_holder/LH = user.get_language_holder()
        if(!LH.has_language(/datum/language/felinid) || !LH.can_speak_language(/datum/language/felinid))
            to_chat(user, "Your mind is filled with the knowledge of huntspeak.")
            LH.grant_language(/datum/language/felinid,TRUE,TRUE,LANGUAGE_CATEARS)


var/datum/language_holder/LH = user.get_language_holder()
    if(LH.has_language(/datum/language/felinid) || LH.can_speak_language(/datum/language/felinid)) //sanity
        to_chat(user, "You lose the keenness in your ears.")
        LH.remove_language(/datum/language/felinid,TRUE,TRUE,LANGUAGE_CATEARS)


Would that satisfy everyone?

File: code/modules/language/felinid.dm (hunk 2 above)
Comment (Anvilman6) [reply code/modules/language/felinid.dm:9]: Thankfully that is just a holdover from the current state of the language so I can just propose a revision to that as well

**Comment by Anvilman6** [comment]
> hisss

I wanted to avoid this one in my setup because it's already a talk verb elsewhere and using it as a Syllable in a language seemed off - just my .02$ on it though

**Comment by JamieD1** [comment]
This is being redone

**Pull request closed without merging**